atspi-common/src/state.rs

```diff
@@ -11,10 +12,11 @@ use zvariant::{Signature, Type};
 /// of an accessibility object.
 #[bitflags]
 #[repr(u64)]
-#[derive(Clone, Copy, Debug, Serialize, Deserialize, PartialEq, Eq)]
+#[derive(Clone, Copy, Debug, Serialize, Deserialize, PartialEq, Eq, Hash, Default)]
```

Comment:
Love `Default`. Hash is here to help users storing states in a `HashMap`?

Reply:
Using a State as a *key* in a HashMap.